src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,90 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { evaluateOnce } from '../derived/evaluateOnce';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  let alwaysNode;
+  let isStarted = false;
+  let isDone = false;
+  let wasStopped = false;
+  return {
+    start: animationCallback => {
+      if (isStarted) {
+        animationCallback && animationCallback({ finished: false });
+        return;
+      }
+      if (isDone) {
+        console.warn('Animation has been finished before');
+        // inconsistent with React Native
+        return;
+      }
+      isStarted = true;
+      evaluateOnce(
+        set(currentState.position, value),
+        currentState.position,
+        () => {
+          alwaysNode = always(
+            set(
+              value,
+              block([
+                cond(clockRunning(newClock), 0, startClock(newClock)),
+                node(newClock, currentState, config),
+                cond(currentState.finished, [
+                  call([], () => {
+                    isStarted = false;
+                    isDone = true;
+                    value.__setAnimation(null, !wasStopped);
+                    if (!wasStopped) {
+                      wasStopped = false;
+                    }
+                  }),
+                  stopClock(newClock),
+                ]),
+                currentState.position,
+              ])
+            )
+          );
+          value.__setAnimation({
+            node: alwaysNode,
+            animationCallback: arg => {
+              animationCallback && animationCallback(arg);
+            },
+          });
+        }
+      );
+    },
+    stop: () => {
+      if (isDone) {
+        console.warn('Animation has been finished before');
+        return;
+      }
+      if (!isStarted) {
+        console.warn("Animation hasn't been started");
+        return;
+      }
+      wasStopped = true;
+      evaluateOnce(set(currentState.finished, 1), currentState.finished);
+    },
+    __stopImmediately_testOnly: () => value.__setAnimation(null, true),
```

Comment:
I'd prefer if we use more logic here that is used outside of the test environment. Specifically we could define the method from line 44 in the main scope (next to `alwaysNode`) and here we could call that method simulating that code which is expected to execute when  we set `finished = 1` is run. We should also call to `stop()` here prior to that